New documentation content (#671)
graknlabs/docs: 0001-typedb/01-start/02-installation.md

opening a new terminal to run TypeDB Console.
</div>

<div class="note">
[Note]
If the following error occurs, please try to install the "C++ redistributable" by following the instructions here.

```shell
Exception in thread "main" java.lang.UnsatisfiedLinkError:
C:\Users\Vaticle\AppData\Local\Temp\ortools-java\win32-x86-64\jniortools.dll: Can't find dependent libraries
```
</div>

### Stop

Press Ctrl-C in the terminal running TypeDB.

## Linux

### Install

#### APT

1. Add the TypeDB repository:
   
   ```bash
   sudo apt install software-properties-common apt-transport-https gpg
   gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-key 8F3DA4B5E9AEF44C
   gpg --export 8F3DA4B5E9AEF44C | sudo tee /etc/apt/trusted.gpg.d/vaticle.gpg > /dev/null
   echo "deb [ arch=all ] https://repo.vaticle.com/repository/apt/ trusty main" | sudo tee /etc/apt/sources.list.d/vaticle.list > /dev/null
   ```

2. Update the package cache:

   ```bash
   sudo apt update
   ```
3. Ensure Java 11+ is installed:

   ```bash
   sudo apt install openjdk-11-jre
   ```

   TypeDB supports the [OpenJDK](https://jdk.java.net) and
   [Oracle JDK](https://www.oracle.com/java/technologies/downloads/).

> 4. Check the latest version number for typedb-server and it’s dependencies:

Reviewer: ```suggestion
4. Check the latest version number for typedb-server and its dependencies:
```
